features/expectation_framework_integration/aggregating_failures.feature

```diff
@@ -59,7 +59,8 @@ Feature: Aggregating Failures
                 # ./spec/use_block_form_spec.rb:18
                 # ./spec/use_block_form_spec.rb:10
 
-                1.1.1) Failure/Error: expect(response.status).to eq(200)
+                1.1.1) Failure/Error:
+                         expect(response.status).to eq(200)
```

Comment:
For multiline snippets it should certainly be started on the next line but for single line snippets I think I prefer it to be on the same line like it was before.  It cuts down on unnecessary churn in this PR if you don't have to change all the expectations about the failure formatting and keeping it to one line means the failure output doesn't take up as much vertical space, which is always nice.

How much effort would it be to keep single-line failures on the same line as `Failure/Error:`?